test/unit/reducers/invoice.spec.js

```diff
@@ -11,6 +13,17 @@ import invoiceReducer, {
   INVOICE_FAILED
 } from 'reducers/invoice'
 
+jest.mock('electron', () => {
```

Comment:
This is probably no longer needed as we have electron mocked by default via `__mocks__` dir